### PR TITLE
sfAutoload include fix for HHVM support

### PR DESCRIPTION
**Pull request by zkiiito**
fixes the 2 dubious tests, hhvm is not likely to fix this issue soon :(

### CODE REVIEW

**Comment by j0k3r** [comment]
Linked to https://github.com/facebook/hhvm/issues/1447

**Comment by j0k3r** [comment]
Now only APC is failing, that's great ! :+1:

File: lib/autoload/sfAutoload.class.php [hunk 1]
```diff
@@ -138,7 +138,15 @@ public function reloadClasses($force = false)
 
     $file = $configuration->getConfigCache()->checkConfig('config/autoload.yml');
 
-    $this->classes = include($file);
+    if (defined('HHVM_VERSION'))
+    {
+      // workaround for https://github.com/facebook/hhvm/issues/1447
+      $this->classes = eval(str_replace('<?php', '', file_get_contents($file)));
```
Comment (j0k3r) [inline lib/autoload/sfAutoload.class.php]: Does `eval` is always enabled in HHVM ?

Comment (zkiiito) [inline lib/autoload/sfAutoload.class.php]: I do not know any other way to disable it than the [disable_functions](http://php.net/manual/en/ini.core.php#ini.disable-functions) ini directive.

Comment (j0k3r) [inline lib/autoload/sfAutoload.class.php]: And `disable_functions` doesn't seems to be part of HHVM for now: https://github.com/facebook/hhvm/issues/2745

Comment (zkiiito) [inline lib/autoload/sfAutoload.class.php]: as reading a bit further, eval is not a function, but a language construct, so it's not even possible to disable with disable_functions.

**Comment by j0k3r** [comment]
Thanks, good job !

**Pull request merged**